Source/Krypton Components/Krypton.Ribbon/View Draw/ViewDrawRibbonGroupClusterButtonText.cs

```diff
@@ -100,7 +100,17 @@ public void MakeDirty()
         /// <param name="context">Layout context.</param>
         public override Size GetPreferredSize([DisallowNull] ViewLayoutContext context)
         {
-            Debug.Assert(context != null);
+            Debug.Assert(context is not null);
+
+            if (context is null)
```

Comment:
Why throw an exception, when the null condition is being dealt with below ?

Reply:
All the DebugAssert statement, throughout the code base, cause [DisallowNull] to be invalidated and the warnings reappear, that X could dereference a possible null value.

using context.Renderer causes warnings without a null check.
```cs
        public override Size GetPreferredSize([DisallowNull] ViewLayoutContext context)
        {
            Debug.Assert(context is not null);

            // Validate incoming reference
            if (context is null)
            {
                throw new ArgumentNullException(nameof(context));
            }

            if (context.Renderer is null)
            {
                throw new ArgumentNullException(nameof(context.Renderer));
            }
```